### ISSUE TITLE
Composer installer not found

### ISSUE THREAD

**Issue by Deele** [geerlingguy/drupal-vm#1901]
### Issue Type

 - Bug Report / Support Request

### Your Environment

```
Vagrant 2.2.3
```

No vboxmanage and ansible not found

### Your OS

 - **Windows** (10)

### Full console output

```
TASK [geerlingguy.composer : Set php_executable variable to a default if not defined.] ***
skipping: [project-lvho-st]

TASK [geerlingguy.composer : Check if Composer is installed.] ******************
ok: [project-lvho-st]

TASK [geerlingguy.composer : Get Composer installer signature.] ****************
ok: [project-lvho-st]

TASK [geerlingguy.composer : Download Composer installer.] *********************
changed: [project-lvho-st]

TASK [geerlingguy.composer : Run Composer installer.] **************************
fatal: [project-lvho-st]: FAILED! => {"changed": false, "cmd": "php composer-installer.php", "msg": "[Errno 2] No such file or directory", "rc": 2}
```

### Summary

Can't `up` vagrant box, it fatally ends with being unable to install composer properly

**Comment by geerlingguy** [comment]
Can you try `vagrant destroy -f` (to delete the VM) then `vagrant up` again to try creating it again? Sometimes network issues may cause the composer installer download to fail or have an issue... maybe that's what happened here.

**Comment by Deele** [comment]
I did that (but without `-f` flag) multiple times and it could not bring it up successfully.

First time, I installed drupal-vm, I did it with empty drupal folder, just to make sure that installation does not ruin my drupal code.

Maybe, it has to do with the fact, that I did not remove it now, and I have it inside existing drupal installation that is made via https://github.com/drupal-composer/drupal-project previously?
